server/views/pages/bookedtoday/summaryMoveOnly.njk

```diff
@@ -4,6 +4,8 @@
 
 {% set pageTitle = "Summary for person booked to arrive" %}
 {% set hasBackLink = false %}
+{% set hasDpsLink = false %}
```

Comment:
is this the wrong way round - it seems like most pages don't have the dps back link?
Would it be better to default to not having the link and only set the variable when you do have it?

Reply:
yes that would make more sense